packages/custom-webpack/examples/full-cycle-app/karma.conf.js

```diff
@@ -1,35 +1,47 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
-module.exports = function(config) {
+const path = require('path');
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+module.exports = function (config) {
   config.set({
     basePath: '',
     frameworks: ['jasmine', '@angular-devkit/build-angular'],
     plugins: [
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
       require('karma-jasmine-html-reporter'),
-      require('karma-coverage-istanbul-reporter'),
+      require('karma-coverage'),
```

Comment:
Could you elaborate on why this is better?

Reply:
Though istanbul reporter requires a payment for support and Angular officially switched to that